api/routers/edr.py

```diff
@@ -78,28 +95,17 @@ async def get_locations(
         )
 
     ts_response = await get_obs_request(ts_request)
+    # logger.info(ts_response.ByteSize())
 
     platform_parameters: DefaultDict[str, Set[str]] = defaultdict(set)
     platform_coordinates: Dict[str, Set[Tuple[float, float]]] = defaultdict(set)
-    all_parameters: Dict[str, Parameter] = {}
     for obs in ts_response.observations:
-        parameter = make_parameter(obs.ts_mdata)
         platform_parameters[obs.ts_mdata.platform].add(obs.ts_mdata.parameter_name)
         # Take last point
         platform_coordinates[obs.ts_mdata.platform].add(
             (obs.obs_mdata[-1].geo_point.lon, obs.obs_mdata[-1].geo_point.lat)
         )
-        # Check for inconsistent parameter definitions between stations
-        # TODO: How to handle those?
-        if obs.ts_mdata.parameter_name in all_parameters and all_parameters[obs.ts_mdata.parameter_name] != parameter:
-            raise HTTPException(
-                status_code=500,
-                detail={
-                    "parameter": f"Parameter with name {obs.ts_mdata.parameter_name} "
-                    f"has multiple definitions:\n{all_parameters[obs.ts_mdata.parameter_name]}\n{parameter}"
-                },
-            )
-        all_parameters[obs.ts_mdata.parameter_name] = parameter
+        # TODO: How to check for inconsistent parameter definitions between stations
```

Comment:
Is this possible to do on lines 76-79? Because if a parameter_name exists that is not unique, it is likely impossible to distinguish between them for stations.